### PR TITLE
feat: report create dashboard type

### PR DESCRIPTION
**Pull request by LoisChen68**

### CODE REVIEW

**Comment by LoisChen68** [comment]
[(2)【報表分析】要可支援放入Dashboard](https://trello.com/c/jZljhTKr/5028-2%E3%80%90%E5%A0%B1%E8%A1%A8%E5%88%86%E6%9E%90%E3%80%91%E8%A6%81%E5%8F%AF%E6%94%AF%E6%8F%B4%E6%94%BE%E5%85%A5dashboard)

File: src/components/report/ReportAdminModal.tsx [hunk 1]
```diff
@@ -1,30 +1,35 @@
...
   {
     report?: ReportProps
     onRefetch?: () => void
+    reports: ReportProps[]
   } & AdminModalProps
-> = ({ report, onRefetch, onCancel, ...props }) => {
+> = ({ report, onRefetch, onCancel, reports, ...props }) => {
+  const originFormType = (report && Object.keys(report.options?.metabase?.resource)[0]) || 'question'
   const [loading, setLoading] = useState(false)
+  const [formType, setFormType] = useState<'question' | 'dashboard'>('question')
```
Comment (Ponchimeow) [inline src/components/report/ReportAdminModal.tsx:32]: replace default value 'question' with `Object.keys(report.options?.metabase?.resource)[0]) || 'question'`

Comment (LoisChen68) [reply src/components/report/ReportAdminModal.tsx]: Thank for review. Modifyed.

**Review by Ponchimeow — APPROVED** [review]

File: src/components/report/ReportAdminModal.tsx (hunk 1 above)
Comment (kkshyu) [reply src/components/report/ReportAdminModal.tsx]: better to change into the following format:
```
options: {
    metabase: {
        type: "dashboard",
        no: 123
    }
}
```

**Pull request merged**